diff --git a/Makefile b/Makefile
index df926f2..ffd26c3 100644
--- a/Makefile
+++ b/Makefile
@@ -8,7 +8,7 @@
 # Pyroute version and release
 #
 version ?= 0.5
-release := $(shell git describe | sed 's/-[^-]*$$//;s/-/.post/')
+release := 0.5.14
 ##
 # Python-related configuration
 #
diff --git a/setup.py b/setup.py
index 5c387f0..b0f1449 100644
--- a/setup.py
+++ b/setup.py
@@ -23,7 +23,7 @@ module = __import__(config.get('setup', 'setuplib'),
                     ['setup'], 0)
 setup = getattr(module, 'setup')
 
-readme = open("README.md", "r")
+readme = open("README.rst", "r")
 
 
 setup(name='pyroute2',
